<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pye</class>
 <widget class="QDialog" name="pye">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos.qrc">
    <normaloff>:/new/prefix1/iconos/probabilidad.png</normaloff>:/new/prefix1/iconos/probabilidad.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="ventanas">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="hd">
      <attribute name="title">
       <string>Histograma</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QTableWidget" name="h">
             <property name="maximumSize">
              <size>
               <width>320</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>C059</family>
               <pointsize>15</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="tabletTracking">
              <bool>false</bool>
             </property>
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="statusTip">
              <string/>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(167, 167, 167);</string>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="textElideMode">
              <enum>Qt::ElideNone</enum>
             </property>
             <property name="gridStyle">
              <enum>Qt::NoPen</enum>
             </property>
             <property name="rowCount">
              <number>1</number>
             </property>
             <property name="columnCount">
              <number>3</number>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <row>
              <property name="text">
               <string>1</string>
              </property>
             </row>
             <column>
              <property name="text">
               <string>No.</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Frecuencia</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>%</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTabWidget" name="tabWidget">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="barras">
                <attribute name="title">
                 <string>Barras</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QFrame" name="baseBarras">
                   <layout class="QHBoxLayout" name="horizontalLayout_10"/>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="pastel">
                <attribute name="title">
                 <string> Pastel</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <widget class="QFrame" name="basePastel">
                   <layout class="QHBoxLayout" name="horizontalLayout_7"/>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="Lineal">
                <attribute name="title">
                 <string>Lineal</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="0" column="0">
                  <widget class="QFrame" name="baseLineal">
                   <layout class="QHBoxLayout" name="horizontalLayout_11"/>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QPushButton" name="MenosZoom">
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="iconos.qrc">
                     <normaloff>:/new/prefix1/iconos/simbolo-menor-que.png</normaloff>:/new/prefix1/iconos/simbolo-menor-que.png</iconset>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="ZoomOriginal">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="iconos.qrc">
                     <normaloff>:/new/prefix1/iconos/marcador-de-posicion.png</normaloff>:/new/prefix1/iconos/marcador-de-posicion.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>30</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="MasZoom">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="iconos.qrc">
                     <normaloff>:/new/prefix1/iconos/mayor-que-el-simbolo.png</normaloff>:/new/prefix1/iconos/mayor-que-el-simbolo.png</iconset>
                   </property>
                   <property name="flat">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="Mayor">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(167, 167, 167);</string>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="text">
                  <string>Mayor: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Media">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(167, 167, 167);</string>
                 </property>
                 <property name="text">
                  <string>Media: </string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="Menor">
                 <property name="text">
                  <string>Menor: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Mediana">
                 <property name="text">
                  <string>Mediana:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="Varianza">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(167, 167, 167);
</string>
                 </property>
                 <property name="text">
                  <string>Varianza: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Moda">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(167, 167, 167);</string>
                 </property>
                 <property name="text">
                  <string>Moda: </string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="DesviacionE">
                 <property name="text">
                  <string>Desviacion E.: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Sumatoria">
                 <property name="text">
                  <string>Sumatoria: </string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="guardar">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="iconos.qrc">
               <normaloff>:/new/prefix1/iconos/excel.png</normaloff>:/new/prefix1/iconos/excel.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ddf">
      <attribute name="title">
       <string>Distribucion de Frecuencias</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTableWidget" name="d">
         <property name="font">
          <font>
           <family>C059</family>
           <pointsize>15</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 15pt &quot;C059&quot;;
background-color: rgb(167, 167, 167);</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>3</number>
         </property>
         <property name="columnCount">
          <number>6</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Clase</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Limites</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Frecuencia</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>F. Acomulada</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>F. Absoluta</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Marca de Clase</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>
